src/Text/Pandoc/Readers/Man.hs

```diff
@@ -0,0 +1,284 @@
+      else return $ Header 1 nullAttr [Str mantitle]
+    where
+    changeTitle title mst@ManState{ pState = pst} =
+      let meta = stateMeta pst
+          metaUp = Meta $ insert "title" (MetaString title) (unMeta meta)
+      in
+      mst { pState = pst {stateMeta = metaUp} }
+
+  macroCodeBlock :: PandocMonad m => Bool -> ManParser m ()
+  macroCodeBlock insideCB = modifyRoffState (\rst -> rst{inCodeBlock = insideCB}) >> return ()
+
+  macroBR :: String -> Bool -> Block
+  macroBR txt inCode | inCode    = Plain [Code nullAttr txt]
+                     | otherwise = fromMaybe (Plain [Strong [Str txt]]) (linkToMan txt)
+
+  linkToMan :: String -> Maybe Block
+  linkToMan txt = case runParser linkParser () "" txt of
+      Right lnk -> Just $ Plain [lnk]
+      Left _ -> Nothing
+    where
+    linkParser :: Parsec String () Inline
+    linkParser = do
+      mpage <- many1 alphaNum
+      space
+      char '('
+      mansect <- digit
+      char ')'
+      -- assuming man pages are generated from Linux-like repository
+      let manurl pagename section = "../"++section++"/"++pagename++"."++section
+      return $ Link nullAttr [Str txt] (manurl mpage [mansect], mpage)
+
+    
+  unkownMacro :: PandocMonad m => String -> ManParser m Block
+  unkownMacro mname = do
+    pos <- getPosition
+    logMessage $ SkippedContent ("Unknown macro: " ++ mname) pos
+    return Null
+   
+  parseArgs :: PandocMonad m => ManParser m [String]
+  parseArgs = do
+    eolOpt <- optionMaybe $ char '\n'
+    if isJust eolOpt
+      then return []
+      else do
+        many1 space
+        arg <- try quotedArg <|> plainArg
+        otherargs <- parseArgs
+        return $ arg : otherargs
+
+    where
+
+    plainArg :: PandocMonad m => ManParser m String
+    plainArg = many1 $ noneOf " \t\n"
+
+    quotedArg :: PandocMonad m => ManParser m String
+    quotedArg = do
+      char '"'
+      val <- many1 quotedChar
+      char '"'
+      return val
+
+    quotedChar :: PandocMonad m => ManParser m Char
+    quotedChar = noneOf "\"\n" <|> try (string "\"\"" >> return '"')
+
+roffInline :: RoffState -> String -> [Inline]
+roffInline rst str
+  | null str && (not $ inCodeBlock rst) = []
+  | inCodeBlock rst = [Code nullAttr str]
+  | otherwise       = case fontKind rst of
+    Regular    -> [Str str]
+    Italic     -> [Emph [Str str]]
+    Bold       -> [Strong [Str str]]
+    ItalicBold -> [Emph [Strong [Str str]]]
+
+parseLine :: PandocMonad m => ManParser m Block
+parseLine = do
+  parts <- parseLineParts
+  newline
+  return $ if null parts
+    then Null
+    else Plain parts
+  where
+    parseLineParts :: PandocMonad m => ManParser m [Inline]
+    parseLineParts = do
+      lnpart <- many $ noneOf "\n\\"
+      ManState {rState = roffSt} <- getState
+      let inls = roffInline roffSt lnpart
+      others <- backSlash <|> return []
+      return $ inls ++ others
+    
+    backSlash :: PandocMonad m => ManParser m [Inline]
+    backSlash = do
```

Comment:
I didn't look at this in detail, but I wasn't sure it handled things like `\f[CB]`, which is valid groff.